changes/README.md

```diff
@@ -0,0 +1,54 @@
+# Fragments for the changelog
+
+This directory contains fragments for the future [NEWS](../NEWS.md) file.
+
+## Introduction
+
+We use <code>[towncrier]</code> to produce useful, summarized news files.
+
+There are 3 sections types:
+
+- API: `changes/api`
+- Tools: `changes/tools`
+- Build System: `changes/build`
+
+There are 3 news fragments types:
+
+- Breaking changes: `.breaking`
+- New: `.feature`
+- Fixes: `.bugfix`
+
+[towncrier]: https://pypi.org/project/towncrier/
+
+## Adding a fragment
+
+Add a short description of the change in a file `changes/SECTION/ID.FRAGMENT`,
```

Comment:
Is it possible to have an `.md` suffix so it renders as markdown in github?

Reply:
Yes, it is accepted. Good idea!